app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+class RecipesController < ApplicationController
+  before_action :set_recipe, only: %i[show edit update destroy]
+  before_action :set_recipe_categories, except: %i[index]
+  before_action :set_ingredients, except: %i[index]
+  before_action :set_appliances, except: %i[index]
+
+  # GET /recipes or /recipes.json
+  def index
+    @recipes = Recipe.all
+  end
+
+  # GET /recipes/1
+  def show; end
+
+  # GET /recipes/new
+  def new
+    @recipe = Recipe.new
+  end
+
+  # GET /recipes/1/edit
+  def edit; end
+
+  # POST /recipes
+  def create
+    @recipe = Recipe.new(recipe_params.merge(steps: sanitize_steps, user_id: current_user.id))
```

Comment:
You are doing this merge below. I believe this merge isn't needed anymore!

Reply:
Merge = gone!